CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# How to Contribute
+
+*eksctl* is [Mozilla Public License Version 2.0 licenced](LICENSE.MPL-2.0) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+# Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. No action from you is required, but it's a good idea to see the
+[DCO](DCO) file for details before you start contributing code to Scope.
+
+# Chat
+
+The project uses Slack. Join the [Weave community](https://weaveworks.github.io/community-slack/) Slack workspace and use the [#eksctl](https://weave-community.slack.com/messages/eksctl/) channel.
+
+## Getting Started
+
+- Fork the repository on GitHub
+- Read the [README](README.md) for getting started as a user and learn how/where to ask for help 
+- If you want to contribute as a developer, continue reading this document for further instructions
+- Play with the project, submit bugs, submit pull requests!
+
+## Contribution workflow
+
+This is a rough outline of how to prepare a contribution:
+
+- Create a topic branch from where you want to base your work (usually branched from master).
+- Make commits of logical units.
+- Make sure your commit messages are in the proper format (see below).
+- Push your changes to a topic branch in your fork of the repository.
+- If you changed code:
+   - add automated tests to cover your changes
+- Submit a pull request to the original repository.
+
+## How to build and run the project
+
+```bash
+make build
+./eksctl get clusters
+```
+
+## How to run the test suite
+
+You can run the unit tests by simply doing
+
+```bash
+make tests
+```
+
+There are integration tests for *eksctl* being developed and more details of how to run them will be included here. You can follow the progress [here](https://github.com/weaveworks/eksctl/issues/151).
+
+# Acceptance policy
+
+These things will make a PR more likely to be accepted:
+
+ * a well-described requirement
+ * tests for new code
+ * tests for old code!
+ * new code and tests follow the conventions in old code and tests
+ * a good commit message (see below)
+
+In general, we will merge a PR once two maintainers have endorsed it.
```

Comment:
Is this true? Who are the two (or more?) designated maintainers right now?

Reply:
I'll change to `we will merge a PR once a maintainer has reviewed and approved it`